.expeditor/scripts/verify/shared.sh

```diff
@@ -52,10 +52,11 @@ install_rust_toolchain() {
   fi
 }
 
-install_rustfmt() {
-  local toolchain="${1?toolchain argument required}"
-  install_rust_toolchain "$toolchain"
-  rustup component add --toolchain "$toolchain" rustfmt
+# Get the version of the nightly toolchain we use for compiling,
+# running, tests, etc.
+get_nightly_toolchain() {
+    dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
```

Comment:
Calculating the relative dir can be done just once at the top of the file and shared between these various functions if you like. For example:
```bash
source_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
project_root="$source_dir/../../.."
```
Simplifying this function to
```bash
    cat "$project_root/RUST_NIGHTLY_VERSION"
```

Again, not necessary for correctness, just a suggestion if you find that more readable/maintainable.